controllers/admin/self-managed/UpdatePageBackupController.php

```diff
@@ -28,84 +28,46 @@
-use PrestaShop\Module\AutoUpgrade\Parameters\UpgradeFileNames;
 use PrestaShop\Module\AutoUpgrade\Router\Routes;
+use PrestaShop\Module\AutoUpgrade\Task\TaskName;
+use PrestaShop\Module\AutoUpgrade\Task\TaskType;
 use PrestaShop\Module\AutoUpgrade\Twig\PageSelectors;
 use PrestaShop\Module\AutoUpgrade\Twig\UpdateSteps;
-use PrestaShop\Module\AutoUpgrade\Twig\ValidatorToFormFormater;
 use Symfony\Component\HttpFoundation\JsonResponse;
 
 class UpdatePageBackupController extends AbstractPageWithStepController
 {
     const CURRENT_STEP = UpdateSteps::STEP_BACKUP;
 
-    protected function getPageTemplate(): string
+    /**
+     * {@inheritdoc}
+     */
+    public function index()
     {
-        return 'update';
+        return $this->redirectTo(Routes::UPDATE_PAGE_BACKUP_OPTIONS);
     }
 
-    protected function getStepTemplate(): string
+    protected function getPageTemplate(): string
     {
         return self::CURRENT_STEP;
     }
 
-    protected function displayRouteInUrl(): ?string
-    {
-        return Routes::UPDATE_PAGE_BACKUP;
-    }
-
-    public function submitBackup(): JsonResponse
-    {
-        $imagesIncluded = $this->upgradeContainer->getUpgradeConfiguration()->shouldBackupImages();
-
-        return $this->displayDialog($imagesIncluded ? 'dialog-backup-all' : 'dialog-backup', [
-            'dialogId' => 'dialog-confirm-backup',
-        ]);
-    }
-
-    public function submitUpdate(): JsonResponse
-    {
-        return $this->displayDialog('dialog-update', [
-            'noBackUp' => !$this->request->request->getBoolean('backupDone', false),
-            'dialogId' => 'dialog-confirm-update',
-
-            'form_route_to_confirm' => Routes::UPDATE_STEP_BACKUP_CONFIRM_UPDATE,
-
-            // TODO: assets_base_path is provided by all controllers. What about a asset() twig function instead?
-            'assets_base_path' => $this->upgradeContainer->getAssetsEnvironment()->getAssetsBaseUrl($this->request),
-        ]);
-    }
-
-    public function startBackup(): JsonResponse
+    protected function getStepTemplate(): string
     {
-        return AjaxResponseBuilder::nextRouteResponse(Routes::UPDATE_STEP_BACKUP);
+        // Different from self::CURRENT_STEP, because a refresh should return to the backup options.
+        return 'backup';
     }
 
-    public function startUpdate(): JsonResponse
+    protected function displayRouteInUrl(): ?string
     {
-        return AjaxResponseBuilder::nextRouteResponse(Routes::UPDATE_STEP_UPDATE);
+        return Routes::UPDATE_PAGE_BACKUP;
     }
 
-    public function saveOption(): JsonResponse
+    public function saveBackupIsCompleted(): JsonResponse
     {
-        $upgradeConfiguration = $this->upgradeContainer->getUpgradeConfiguration();
-        $upgradeConfigurationStorage = $this->upgradeContainer->getUpgradeConfigurationStorage();
+        $this->upgradeContainer->getState()->setBackupCompleted(true);
```

Comment:
it should be put here https://github.com/PrestaShop/autoupgrade/blob/a7060e70a7e3aefdb2e72bd8a954c92d4549caa6/classes/Task/Backup/BackupComplete.php#L57-L56